dashboard/db/schema.rb

```diff
@@ -714,12 +714,12 @@
     t.string "level_num"
     t.bigint "ideal_level_source_id", unsigned: true
     t.integer "user_id"
-    t.text "properties", limit: 16777215
+    t.text "properties", limit: 4294967295, collation: "utf8mb4_unicode_520_ci"
     t.string "type"
     t.string "md5"
     t.boolean "published", default: false, null: false
-    t.text "notes"
-    t.text "audit_log"
+    t.text "notes", limit: 16777215
+    t.text "audit_log", limit: 16777215
```

Comment:
hi @megcrenshaw , sorry, but this looks wrong -- you're introducing changes to schema.rb which will not be generated by your migration, meaning other developers will end up with schema diffs (and diffs in all the level models) after they migrate their database next. to fix this, please run these commands on your local db:
```
mysql> ALTER TABLE levels MODIFY notes text;
mysql> ALTER TABLE levels MODIFY audit_log text;
```
then commit the changes and send that for review.

Reply:
Dang, you're right. Here's what I've done, but still no schema changes:
1) Checked out a new branch from staging
2) Ran rake db:reset
3) Ran rake db:migrate
4) Ran the commands you listed in SQL above
`git status` doesn't show any changes ... not sure. I've got to run for the day, but can investigate further tomorrow. Thanks for the heads up

Reply:
it looks like you got this sorted 👍